wdl-grammar/src/v1/lint/preamble_comment.rs

```diff
@@ -72,6 +66,18 @@ impl<'a> Rule<&Pair<'a, v1::Rule>> for PreambleComment {
         TagSet::new(&[lint::Tag::Style])
     }
 
+    fn body(&self) -> &'static str {
+        "Preamble comments are full line comments before the version declaration and they start \
+         with a double pound sign. These comments are reserved for documentation that doesn't fit \
+         within any of the WDL-defined documentation elements (such as `meta` and `parameter_meta` \
+         sections). They may provide context for a collection of tasks or structs, or they may \
+         provide a high-level overview of the workflow. Double-pound-sign comments are not allowed \
+         after the version declaration. All comments before the version declaration should start \
+         with a double pound sign (or if they are not suitable as preamble comments they should be \
+         moved to _after_ the version declaration). Comments beginning with 3 or more pound signs \
+         are permitted after the version declaration, as they are not considered preamble comments."
```

Comment:
I committed this as "first draft" I wanted to revise. But I'm ok with most of it on review. One issue though:

IMO `Comments beginning with 3 or more pound signs are permitted after the version declaration, as they are not considered preamble comments.` implies that triple+ comments are invalid before the version declaration. That is not currently checked by this rule or any other. Over Slack (I think) we agreed that the triple+ case should be warned on. Does that check belong within this rule?
@adthrasher  @peterhuene

Reply:
I think it would belong in this rule.

Reply:
Done and done. Requesting re-review please.